api/polling/README.md

```diff
@@ -1,60 +1,127 @@
 ---
 description: >-
-  How to use the polling pattern.
+  Implementation guide for the polling pattern in Tradecloud API v2
 ---
 
-# Polling usage
+# Polling Pattern Implementation Guide
 
-The polling API is an alternative for the webhook API. The pro's and cons are explained in:
+The polling API offers an alternative to the webhook API for retrieving order updates. Both approaches have different advantages and considerations:
 
 {% page-ref page="../webhook-vs-polling.md" %}
 
-This page explains the polling pattern usage, which consists of 3 steps:
+## Overview
+
+The polling pattern consists of three essential steps:
+
+1. **Fetch** - Retrieve orders or shipments changed since the last poll
+2. **Process** - Handle the retrieved new or updated data
+3. **Persist** - Store the latest timestamp for the next polling cycle
+
+## Implementation steps
+
+### Step 1: Fetch updated data periodically
+
+Set up a scheduled process to fetch all orders or shipments that have been created or changed since your last poll.
+
+#### Polling best practices
+
+- Use a consistent polling interval (typically 5 minutes)
+- Include the `lastUpdatedAt` from previous poll response as the `lastUpdatedAfter` filter
+- Set `limit=100` (maximum allowed value) to control response size
+- Handle pagination when needed:
+  - If response contains `total` > 100, not all updates were returned
+  - Either decrease polling frequency (recommended) or use `offset` parameter for paging
```

Comment:
Small mix up. I assume you mean that the user should poll more often?
Reword it to either:
- Increase the polling frequency
- Decrease the polling period